columnflow/columnar_util.py

```diff
@@ -1553,7 +1559,12 @@ def get_dependencies(self, include_others: bool = False) -> set[ArrayFunction |
 
         return deps
 
-    def _get_columns(self, io_flag: IOFlag, call_cache: set | None = None, debug=False) -> set[str]:
+    def _get_columns(
+        self,
+        io_flag: IOFlag,
+        call_cache: set | None = None,
+        no_dependencies: bool = False,
```

Comment:
There are a other methods that have an argument that control whether certain actions should be taken also for dependencies. They are named a bit inconsistently though (deps, dependencies, recursive, ..). Could you use `dependencies: bool = True` here?

I'll prepare another PR then to adjust the other locations in existing code.

Reply:
That makes sense, I can adjust that :+1:

Reply:
Done